apps/controllerx/cx_devices/aqara.py

```diff
@@ -572,6 +572,114 @@ def get_z2m_actions_mapping(self) -> DefaultActionsMapping:
         }
 
 
+class WXCJKG13LMMediaPlayerController(MediaPlayerController):
+    def get_z2m_actions_mapping(self) -> DefaultActionsMapping:
+        return {
+            "button_1_single": "click_volume_down",
```

Comment:
Use the constants from `apps/controllerx/cx_const.py` file. In this case, it would be `MediaPlayer.CLICK_VOLUME_DOWN`. The same applies to the others.

Reply:
Done.